guides/RELEASE-PROCESS.md

```diff
@@ -130,3 +130,16 @@ Example changes:
 
 - Renamed or removed classes, mixins, functions, placeholders, or global variables.
 - Major visual changes to existing components
+
+# Sketch Library Release Process
+
+1. Download the [Sketch file from GitHub](https://github.com/CMSgov/design-system/blob/master/design-assets/CMS-Design-System-UI-kit.sketch)
+1. Update the CMSDS Sketch file locally with your changes
+1. Update the .xml file in the following ways
+   - Update the item `title` with the version number
+     - Example: `CMS Design System UI kit - v1.1`
```

Comment:
Should this have guidance on how to number versions?

Reply:
We're still working this out